composables/github.ts

```diff
@@ -0,0 +1,35 @@
+export const useGithubCookie = () => useCookie('gh_token')
```

Comment:
This composable is mostly semantic. Ensuring all cookie usages use same `gh_token` key and allow later refactor easier to update one place